core/src/main/java/org/apache/carbondata/core/statusmanager/FileFormat.java

```diff
@@ -56,6 +56,13 @@ public static FileFormat getByOrdinal(int ordinal) {
     return COLUMNAR_V3;
   }
 
+  public boolean isCarbonFormat() {
+    if (this.format.equalsIgnoreCase("COLUMNAR_V3") || this.format.equalsIgnoreCase("ROW_V1")) {
```

Comment:
simplify this, change to `return this.format.equalsIgnoreCase("COLUMNAR_V3") | this.format.equalsIgnoreCase("ROW_V1")`

Reply:
Done